lib/web_ui/test/text/line_breaker_test_helper.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.10
+
+const Map<String, String> _replacements = <String, String>{
+  // This following test cases contradict rule LB25, so we are replacing them
```

Comment:
Did you mean "These"? Also, if this comment applies to the entire map, consider moving it up to the declaration level.

Reply:
Good catch, yes it should be "These".